plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/VspResourceCommand.java

```diff
@@ -79,4 +79,41 @@ public String getProxyUserDomainuuid() {
     public boolean executeInSequence() {
         return false;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
This ``equals`` implementation should include attributes from the ``Command`` super class.